bigbluebutton-html5/imports/ui/components/whiteboard/component.jsx

```diff
@@ -190,7 +190,18 @@ export default function Whiteboard(props) {
               persistShape(shape, whiteboardId);
             }
           });
-    removeShapes(deletedShapes, whiteboardId);
+
+    //order the ids of shapes being deleted to prevent crash when removing a group shape before its children
+    const orderedDeletedShapes = [];
+    deletedShapes.forEach(eid => {
+      if (shapes[eid]?.type !== 'group') {
+        orderedDeletedShapes.unshift(eid);
+      } else {
+        orderedDeletedShapes.push(eid)
+      }
+    });
+
+    removeShapes(orderedDeletedShapes, whiteboardId);

```

Comment:
Also, I noticed we are sending empty removeAnnotations messages to akka when the list is empty here (probably it was already like this before), so it's better to add a check if it's not empty